code/modules/dungeons/junk_tractor_beam.dm

```diff
@@ -99,6 +100,12 @@
 	var/list/preloaded_25_25 = list()  // Need to preload maps in SOUTH direction
 	var/list/preloaded_5_5 = list()  // Need to preload maps in SOUTH direction
 
+	var/list/affinities = list(  // Put them in the same order than the affinities of /datum/junk_field
+		/datum/map_template/junk/j25_25/neutral,
```

Comment:
Here as an associative list with these datums and defines so you wouldn't need to make sure everything is in the same order, just a suggestion though. (2/2)